module-2/backend/week_one.md

```diff
@@ -5,30 +5,57 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+GET - retrive web page, POST - post info to a web page, DELETE - delete a record, UPDATE - update a record, PUT - update or create a record
 2. What is Sinatra?
+A framework for creating web applications
 4. What is MVC?
+Model view and controller model.  This is used to provide a paradigm for designing web applications that interact with databases.
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+To make code easier to understand and maintain
 6. What types of variables are accessible in our view templates without explicitly passing them?
+Instance variables
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
   get '/horses' do
+    @count = 1
     erb :index
   end
   ```
 
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed` to the view?
+ :locals => :name => 'Jane'
 9. What's the purpose of ERB?
+ It allows you to embed ruby code within HTML documents
 10. Why do I need a development AND test database?
+ So you have a secondary database with the same schema to put temporary test data into.
 11. What's responsive design?
+ To make a website responsive changing resolutions and window sizes
 12. What is CRUD and why is it important?
+  CREATE READ UPDATE DELETE - It is used to refer to a web application that interacts with a database
 13. What does HTTP stand for? 
+  HYPERTEXT TRANSFER LANGUAGE
 14. What are the two ways to interpolate Ruby in an ERB view template? What's the difference between these two ways?
+ "<= ruby.object %=>" and <= ruby.object %=>

```

Comment:
<%= ruby.object %> and <% ruby.object %> the first shows visible on the page and the other doesnt show on the page but allows us to manipulate data on an erb template